File: Examples/CSharp/Load-Save-Convert/ExportToSWF.cs
using System.IO;
using Aspose.Diagram;
namespace Aspose.Diagram.Examples.CSharp.Diagrams
{
    public class ExportToSWF
    {
        public static void Run()
        {

            try
            {
                // ExStart:ExportToSWF
                // The path to the documents directory.
                string dataDir = RunExamples.GetDataDir_Diagrams();
                // Load diagram
                Diagram diagram = new Diagram(dataDir + "ActvDir.vsd");
                // Save diagram
                diagram.Save(dataDir + "Output_Out.swf", SaveFileFormat.SWF);
                // ExEnd:ExportToSWF
            }
            catch (System.Exception ex)
            {
                System.Console.WriteLine(ex.Message + "\nThis example will only work if you apply a valid Aspose License. You can purchase full license or get 30 day temporary license from http:// Www.aspose.com/purchase/default.aspx.");
            }
        }
    }
}

File: Examples/VisualBasic/Load-Save-Convert/ExportToSWF.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Diagram

Namespace Diagrams
    Public Class ExportToSWF
        Public Shared Sub Run()
            Try
                ' ExStart:ExportToSWF
                ' The path to the documents directory.
                Dim dataDir As String = RunExamples.GetDataDir_Diagrams()
                ' Load diagram
                Dim diagram As New Diagram(dataDir & "ActvDir.vsd")
                ' Save diagram
                ' ExEnd:ExportToSWF
                diagram.Save(dataDir & "Output_Out.swf", SaveFileFormat.SWF)
            Catch ex As System.Exception
                System.Console.WriteLine("This example will only work if you apply a valid Aspose License. You can purchase full license or get 30 day temporary license from http://www.aspose.com/purchase/default.aspx.")
            End Try
        End Sub
    End Class
End Namespace